Felienne/hedy
# Improve error message when an list needs to be translated #1135

Improve error message when an list needs to be translated #1135
- left, right, number -> left, right or a number
- left, right, number -> left, right of een getal

**Description**


**Fix for**

#1135 

**How to test**

Try the following commands on levels 1 and 2 and in English and Nederlands:
`turn lalala`
`forward lalala`
The expected response is "left, right or a number" instead of "left, right, number"
